apis/v1alpha2/validation/gateway.go

```diff
@@ -44,7 +46,32 @@ func validateGatewaySpec(spec *gatewayv1a2.GatewaySpec, path *field.Path) field.
 // to the Gateway API specification.
 func validateGatewayListeners(listeners []gatewayv1a2.Listener, path *field.Path) field.ErrorList {
 	// TODO [danehans]: Add additional validation of listener fields.
-	return validateListenerHostname(listeners, path)
+	var errs field.ErrorList
+	e1:=validateListenerTLSConfig(listeners, path)
+	errs = append(errs,e1...)
+	e2:=validateListenerHostname(listeners, path)
+	errs = append(errs,e2...)
```

Comment:
Tiny Nit:
```suggestion
	e1 := validateListenerTLSConfig(listeners, path)
	errs = append(errs, e1...)
	e2 := validateListenerHostname(listeners, path)
	errs = append(errs, e2...)
```

Reply:
This same general comment applies through most of this code, essentially to add spaces:

- after commas
- before and after `:=`

Reply:
Fixed this . Do you have any linter for this ? I did go fmt before submitting the patch . I am new to go here :)

Reply:
> Fixed this . Do you have any linter for this ? I did go fmt before submitting the patch . I am new to go here :)

`make verify` will run more linters